### PR TITLE
Test system metrics collector

### PR DESCRIPTION
**Pull request by girikuncoro**
This PR is part of #36, introducing test to all system metrics subcollectors.
    * Test collect cluster status metrics
    * Test collect cluster node metrics
    * Test collect cluster service metrics

Signed-off-by: Giri Kuncoro <girikuncoro@gmail.com>

### CODE REVIEW

File: collector/system_collector.go [hunk 1]
```diff
@@ -62,63 +72,86 @@ func (sc *systemCollector) Describe(ch chan<- *prometheus.Desc) {
...
 	clusterStatus, err := sc.systemClient.ReadClusterStatus()
 	if err != nil {
 		level.Error(sc.logger).Log("msg", "Unable to collect cluster status")
 		return
 	}
-
+	systemStatusMetric := systemStatusMetric{
+		Status: 0.0,
+	}
 	if strings.ToUpper(clusterStatus.ControlClusterStatus.Status) == "STABLE" &&
```
Comment (walbertus) [inline collector/system_collector.go:100]: Just wondering, is this "STABLE" equal with all Cluster Node status "CONNECTED"?

Comment (walbertus) [inline collector/system_collector.go:100]: Just wondering, is this "STABLE" equal with all Cluster Node status "CONNECTED"?

**Review by walbertus — APPROVED** [review]
Looks complicated but all good

File: collector/system_collector.go (hunk 1 above)
Comment (girikuncoro) [reply collector/system_collector.go:100]: I think from what I understand:
STABLE means the controller/manager cluster is healthy as a whole cluster

for each of node that forms the cluster, CONNECTED means whether the particular node is healthy part of the cluster

might need to read the API docs more, but I agree the Enums are varied and confused us

**Pull request merged**